applications/skyepass.md

```diff
@@ -0,0 +1,251 @@
+# Open Grant Proposal
+
+> This document is referenced in the terms and conditions and therefore needs to contain all the required information. Don't remove any of the mandatory parts presented in bold letters or as headlines! See the [Open Grants Program Process](https://github.com/w3f/Open-Grants-Program/blob/master/README_2.md) on how to submit a proposal.
+
+* **Project Name:** SkyePass
+* **Team Name:** SkyeKiwi Team
+* **Payment Address**: 0xa5E4E1BB29eE2D16B07545CCf565868aE34F92a2
+
+*The above combination of your GitHub account submitting the application and payment address will be your unique identifier during the program. Please keep them safe.*
+
+## Project Overview :page_facing_up: 
+
+SkyePass is a decentralized and customizable identity management software. On the surface, it is a decentralized, open source and modern password manager. 
+
+### Product Details
+
+As a long-term password manager software user myself, I have been really frustrated of the services like LastPass, 1Password for either lack of functionalities or the idea of storing ones entire digital identity on their corporate servers. Existing open-source solutions are too technically complicated to use. 
+
+At the very basis of it, a password manager is no more than an encrypted database, an APP and a browser extension to interact with the database. 
+
+Therefore, our team create a new password manger software that has pretty and intuitive UI/UX, fully decentralized (i.e. our team own no backend servers) and hackable by providing an open API for people to develop extensions with. 
+
+Users who signup will first create a blockchain wallet and have the mnemonic (and a master password) as their sole identity credentials (pretty standard blockchain wallet stuff). Later, each database instance is called a `vault` (standard name for all password managers) and they are light-weight file based databases ([lowDB](https://github.com/typicode/lowdb) seems to be a great choice). User can be given options to choose the encryption behavior of their database. By default, the vault will be split into some pieces with a Shamir's secret sharing mechanism. 
```

Comment:
I don't understand this bit. How many vaults do I create? One for each combination of people I want to share something with? 

Will there be an easy way to copy/move a password to a shared vault, similar to how I would "share" a password on Lastpass?

Reply:
Yes. One for each combination of people you wanna share with. It's an idea similar to the team vault and family vault of 1Password. 

It's a compound questions: 1. Yes, there is a way to move/copy a password between vaults (shared or private). It's a matter of changing some local databases. We will try to make the UX as smooth as possible. 2. By matching the experience of sharing a password as Lastpass, correct me if I'm wrong, you are referring to sharing a single item of password not a `vault`: the client will contact another smart contract(smart contract 2) to generate a shared secret of just `one item`, with some expiration time (i.e. after 200 blocks). You can argue either that a `vault` is an array of `single items`, or if `a single item` is a minimized `vault`. After all, there is no big difference. So yes, there will be an easy way to move/copy and to share a single password item.